### ISSUE TITLE
Iframe with 3D transform, content not clipped

### ISSUE THREAD

**Issue by paulrouget** [servo/servo#11650]
Even though both https://github.com/servo/servo/issues/11151 and https://github.com/servo/servo/issues/11358 are now fixed, we still have issues with iframe content not being properly clipped.

Both previous test case render correctly, so it's something new.

Working on a reduced test case now.

<img width="834" alt="screen shot 2016-06-07 at 07 51 46" src="https://cloud.githubusercontent.com/assets/373579/15847258/b61a8ffc-2c84-11e6-97a4-966d3b8cfdb9.png">

**Comment by paulrouget** [comment]
It's 3D transform specific:

```html
<style>
  div, iframe {
    width: 100vw;
    height: 100vh;
    top: 0px;
    left: 0px;
  }
  div {
    perspective: 1000px;
    background: grey;
  }
  iframe {
    display: block;
    transform-origin: left center 0px;
    transform: translate3d(0px, 0px, -200px);
  }
</style>

<div>
  <iframe src="https://duckduckgo.com/?q=foobar&amp;ia=about">
  </iframe>
</div>
```

@pcwalton any ideas? If that's too hard to fix, I think we can simply move to 2D transforms in browserhtml.

**Comment by pcwalton** [comment]
I think this is probably something related to DDG's specific use of stacking contexts, similar to #11648.

**Comment by ConnorGBrewster** [comment]
This seems to be fixed with WR2 #12711
